ui/components/ui/page-container/page-container.component.js

```diff
@@ -74,9 +74,11 @@ export default class PageContainer extends PureComponent {
     children = children.filter(Boolean);
     const { activeTabIndex } = this.state;
 
-    return children[activeTabIndex]
-      ? children[activeTabIndex].props.children
-      : children.props.children;
+    if (Array.isArray(children)) {
```

Comment:
I think line 74 makes this check redundant.

Reply:
Good shout, will look again.